packages/wdio-cli/src/commands/config.ts

```diff
@@ -77,11 +77,18 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
                     target: 'ES5',
                 }
             }
-
-            await fs.promises.writeFile(
-                path.join(process.cwd(), 'tsconfig.json'),
-                JSON.stringify(config, null, 4)
-            )
+            if (!hasFile('tsconfig.json')) {
+                await fs.promises.writeFile(
+                    path.join(process.cwd(), 'tsconfig.json'),
+                    JSON.stringify(config, null, 4)
+                )
+            } else {
+                fs.ensureDirSync(path.join(process.cwd(), 'test'))
+                await fs.promises.writeFile(
+                    path.join(process.cwd(), 'test/tsconfig.json'),
+                    JSON.stringify(config, null, 4)
+                )
+            }
```

Comment:
we can simplify this via:

```suggestion
            await fs.promises.writeFile(
                path.join(process.cwd(), (hasFile('tsconfig.json') ? 'test' : ''), 'tsconfig.json'),
                JSON.stringify(config, null, 4)
            )
```